src/Pact/Native/Ops.hs

```diff
@@ -14,8 +14,14 @@
 
 
 module Pact.Native.Ops
-    (opDefs)
-    where
+    ( opDefs
```

Comment:
Not sure about just exporting all of these.

Reply:
going to defer to @slpopejoy here

Reply:
Why are these being exported?